src/cli.rs

```diff
@@ -381,8 +385,12 @@ Example: --print=Hb"
     ///
     /// A leading colon works as shorthand for localhost. ":8000" is equivalent
     /// to "localhost:8000", and ":/path" is equivalent to "localhost/path".
-    #[clap(value_name = "[METHOD] URL")]
-    raw_method_or_url: String,
+    #[clap(
+        value_name = "[METHOD] URL",
+        required_unless_present = "generate",
+        conflicts_with = "generate"
```

Comment:
```suggestion
        required = true,
```
If we put `required = true` it'll still work and it won't be marked as optional in the completions.

Reply:
Nice! This also seems to fix the generated usage text:

- **Before**: `Usage: xh [OPTIONS] [[METHOD] URL] [REQUEST_ITEM]...`
- **After**: `Usage: xh [OPTIONS] <[METHOD] URL> [REQUEST_ITEM]...`

Technically, it should be `xh [OPTIONS] [METHOD] URL [REQUEST_ITEM]...` but I think we decided on ` xh [OPTIONS] <[METHOD] URL> [REQUEST_ITEM]...` before due to some [limitations](https://github.com/clap-rs/clap/issues/2122) with `clap-rs`.

Reply:
Confirmed this is better, thanks! Incorporating into next commit.